### ISSUE TITLE
SA1648 false positive when implementing static interface members

### ISSUE THREAD

**Issue by bruno-brant** [DotNetAnalyzers/StyleCopAnalyzers#3736]
As the title says - SA1648 is triggered if we add a <inheritdoc /> in a static member that is implementing an interface:

```cs
interface IFoo
{
	/// <summary>Foobles the bars.</summary>
	public static abstract void Bar();
}

class Foo : IFoo
{
	// Line below triggers SA1648
	/// <inheritdoc/>
	public static void Bar() {}
}
```

**Comment by bjornhellander** [comment]
Duplicate of #3595. Already fixed, but not yet released.

**Comment by sharwell** [comment]
Duplicate of #3595
